### PR TITLE
[FIX] account_voucher: fix missing payment_id in "Pay Directly" voucher

### PR DESCRIPTION
**Pull request by smetl**
Create an accout.voucher with "Pay Directly" as payment.
You get something like:

|| Debit | Credit |
---------| ------------- | ------------- |
Expenses| x  |     |
Cash|     | x  |

However, the expense line doesn't appear in the reconciliation widget
because the payment_id is missing in the account.move.line and then, doesn't match
the search domain for the reconciliation.

To solve it, a payment_id is created for this line in case of "Pay Directly" payment.

-opw: 775167

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account_voucher/models/account_voucher.py [hunk 1]
```diff
@@ -222,6 +227,20 @@ def _convert_amount(self, amount):
...
+            'payment_type': 'outbound',
+            'payment_method_id': self.env.ref('account.account_payment_method_manual_out').id,
+            'partner_type': 'supplier',
+            'partner_id': self.partner_id.id,
+            'amount': self.amount,
+            'currency_id': self.currency_id.id,
+            'payment_date': self.date,
+            'journal_id': self.journal_id.id,
+            'company_id': self.company_id.id,
+        }
```
Comment (qdp-odoo) [inline addons/account_voucher/models/account_voucher.py]: missing state in 'posted'

**Comment by qdp-odoo** [comment]
mhm Laurent, I believe you forgot my comments 

>I think that the payment_id should be linked on all the account.move.line of the voucher.
>It would be better also set the payment_reference field

**Comment by smetl** [comment]
Are you sure? Because in my mind, the payment is only originator of the cash move line and this one should be the only line displayed in the reconciliation widget. Is it wrong?

**Comment by qdp-odoo** [comment]
closed by https://github.com/odoo/odoo/commit/e69fb3f82d386b14ffbdc6fda4bb38df533d29b0

**Pull request closed without merging**